ros2node/ros2node/verb/list.py

```diff
@@ -42,6 +42,6 @@ def main(self, *, args):
         elif node_names:
             sorted_names = sorted(n.full_name for n in node_names)
             if has_duplicates(sorted_names):
-                print('WARNING: Be aware that are nodes in the graph that share an exact name, '
-                      'this can have unintended side effects.', file=sys.stderr)
+                print('WARNING: Be aware that there are nodes in the graph that share an exact '
+                      'name, this can have unintended side effects.', file=sys.stderr)
```

Comment:
If we are going to be pedantic about the wording:

```suggestion
                print('WARNING: Be aware that there are nodes in the graph that share an exact '
                      'name, which can have unintended side effects.', file=sys.stderr)
```